tests/integration/targets/mso_schema_site_anp_epg_bulk_staticport/tasks/main.yml

```diff
@@ -0,0 +1,580 @@
+  mso_schema_site_anp_epg_bulk_staticport: 
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: '{{ mso_site | default("ansible_test") }}'
+    template: Template 1
+    anp: AP1
+    epg: EPG1
+    pod: pod-1
+    leaf: 101
+    path: eth1/1
+    vlan: 126
+    mode: native
+    type: port
+    deployment_immediacy: immediate
+    static_ports:
+      - path: eth1/2
+        leaf: 102
+      - path: eth1/3
+        vlan: 124
+    state: present
+  check_mode: true
+  register: cm_add_stat1e1
+
+- name: Verify cm_add_stat1e1
+  assert:
+    that:
+    - cm_add_stat1e1 is changed
+    - cm_add_stat1e1.previous == {}
+    - cm_add_stat1e1.current.epgs[0].staticPorts[0].deploymentImmediacy == 'immediate'
+    - cm_add_stat1e1.current.epgs[0].staticPorts[0].portEncapVlan == 126
+    - cm_add_stat1e1.current.epgs[0].staticPorts[0].path == 'topology/pod-1/paths-102/pathep-[eth1/2]'
+    - cm_add_stat1e1.current.epgs[0].staticPorts[0].mode == 'native'
+    - cm_add_stat1e1.current.epgs[0].staticPorts[0].type == 'port'
+    - cm_add_stat1e1.current.epgs[0].staticPorts[1].deploymentImmediacy == 'immediate'
+    - cm_add_stat1e1.current.epgs[0].staticPorts[1].portEncapVlan == 124
+    - cm_add_stat1e1.current.epgs[0].staticPorts[1].path == 'topology/pod-1/paths-101/pathep-[eth1/3]'
+    - cm_add_stat1e1.current.epgs[0].staticPorts[1].mode == 'native'
+    - cm_add_stat1e1.current.epgs[0].staticPorts[1].type == 'port'
+
+
+- name: Add static port 1 to site EPG1 of AP1 (normal mode)
+  mso_schema_site_anp_epg_bulk_staticport: &add_static_port_1
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: '{{ mso_site | default("ansible_test") }}'
+    template: Template 1
+    anp: AP1
+    epg: EPG1
+    pod: pod-1
+    leaf: 101
+    path: eth1/1
+    vlan: 126
+    mode: native
+    type: port
+    deployment_immediacy: immediate
+    static_ports:
+      - path: eth1/2
+        leaf: 102
+      - path: eth1/3
+        vlan: 124
+    state: present
+  register: nm_add_stat1e1
+
+- name: Verify nm_add_stat1e1
+  assert:
+    that:
+    - nm_add_stat1e1 is changed
+    - nm_add_stat1e1.previous == {}
+    - nm_add_stat1e1.current.epgs[0].staticPorts[0].deploymentImmediacy == 'immediate'
+    - nm_add_stat1e1.current.epgs[0].staticPorts[0].portEncapVlan == 126
+    - nm_add_stat1e1.current.epgs[0].staticPorts[0].path == 'topology/pod-1/paths-102/pathep-[eth1/2]'
+    - nm_add_stat1e1.current.epgs[0].staticPorts[0].mode == 'native'
+    - nm_add_stat1e1.current.epgs[0].staticPorts[0].type == 'port'
+    - nm_add_stat1e1.current.epgs[0].staticPorts[1].deploymentImmediacy == 'immediate'
+    - nm_add_stat1e1.current.epgs[0].staticPorts[1].portEncapVlan == 124
+    - nm_add_stat1e1.current.epgs[0].staticPorts[1].path == 'topology/pod-1/paths-101/pathep-[eth1/3]'
+    - nm_add_stat1e1.current.epgs[0].staticPorts[1].mode == 'native'
+    - nm_add_stat1e1.current.epgs[0].staticPorts[1].type == 'port'
+
```

Comment:
Can you add static port 1 to site EPG1 of AP1 (normal mode) again and check for idempotency?
It would be great if you could do it after adding each element in the other tests.